### PR TITLE
NewRoutineView first PR

### PR DESCRIPTION
**Pull request by YLee2121**
# NewRoutineView first PR

## 💪🏿NewRoutineView

* `newRoutineButton` with a navLink to `RoutineSelectionView()`

## 😒PS

* `.gitattributes` is temporarily existing for safer PR.

* Add the `.pbxproj` to `.gitignore` also for safer PR.

## 📷Screenshots

* `NewRoutineView` 
image image

* `NewRoutineView` with `MainView`
image  image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -132,3 +132,4 @@ iOSInjectionProject/
 **/xcshareddata/WorkspaceSettings.xcsettings
 
 # End of https://www.toptal.com/developers/gitignore/api/macos,xcode,swift
+GymBro.xcodeproj/project.pbxproj
```
Comment (YLee2121) [inline .gitignore]: Delete the line

File: .gitignore [hunk 2]
```diff
@@ -131,4 +131,4 @@ iOSInjectionProject/
 /*.gcno
 **/xcshareddata/WorkspaceSettings.xcsettings
 
-# End of https://www.toptal.com/developers/gitignore/api/macos,xcode,swift
```
Comment (alex308248) [inline .gitignore:134]: Having another PR to change the .gitginore file will be safter, but it doesn't actually matter in this case.

**Review by alex308248 — APPROVED** [review]
Viewed. Just make sure the commits will be squashed before merging : )

File: .gitignore (hunk 2 above)
Comment (YLee2121) [reply .gitignore:134]: Will do it next time

**Comment by cheniwei** [comment]
Looks better! Done review.

**Review by cheniwei — APPROVED** [review]

**Comment by alex308248** [comment]
Don't forget to pull the latest version1 before merging~~

**Pull request merged**